docs/setup.md

```diff
@@ -183,18 +186,52 @@ Development mode will target the latest versions of Chrome, Firefox and Safari f
 
 The Rails server will start on <http://localhost:3000>.
 
+To sign in locally, you can use any username and password from your local database. The seed task creates users in various roles all with the password `REVIEW_APP_USER_PASSWORD` from your .env file (which has default value `password`).
+You can also create a user in the database manually with the details of your choice, but you will need to generate a correctly encrypted password. You can find the value to use for encrypted password which corresponds to the password `YOURPASSWORDHERE` using `User.new(:password => [YOURPASSWORDHERE]).encrypted_password`.
```

Comment:
You could add a user through rails console by running something similar to the commands we have in `seeds.rb` as well. That would allow you to specify non encrypted password. Not sure if it's worth adding that. Or is that what you mean here by creating the user in the db manually?